engine/src/tangl/story38/__init__.py

```
@@ -0,0 +1,37 @@
+    InitReport,
+    StoryCompiler38,
+    StoryInitResult,
+    StoryMaterializer38,
+    StoryTemplateBundle,
+    UnresolvedDependency,
+    World38,
+)
+from .story_graph import StoryGraph38
+
+# Register story-level journal handlers.
+from . import system_handlers  # noqa: F401
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)